.github/workflows/build_and_push_to_docker.yml

```diff
@@ -62,7 +62,7 @@ jobs:
 
         # This ensures the same image tags as before are built, in addition to new ones
         if [[ "${{ matrix.BASE_IMAGE_TAG }}" == lab-* ]]; then
-          export LABEL_PREFIX=3.9
+          export LABEL_PREFIX=3.10
```

Comment:
@rokroskar I thought none of the images are using python 3.10. And from Ralf's comment it seems we are not ready for 3.10 yet.

So this should then probably be rolled back to 3.9?

Reply:
```suggestion
          export LABEL_PREFIX=3.9
```